custom_sessions/models/base/snowplow_base_events_this_run.sql

```diff
@@ -0,0 +1,25 @@
+{#
+Copyright (c) 2023-present Snowplow Analytics Ltd. All rights reserved.
+This program is licensed to you under the Snowplow Community License Version 1.0,
+and you may not use this file except in compliance with the Snowplow Community License Version 1.0.
+You may obtain a copy of the Snowplow Community License Version 1.0 at https://docs.snowplow.io/community-license-1.0
+#}
+
+{%- set lower_limit, upper_limit = snowplow_utils.return_limits_from_model(ref(var('snowplow__base_sessions', 'snowplow_base_sessions_this_run')),
+                                                                            'start_tstamp',
+                                                                            'end_tstamp') %}
+
+{% set base_events_query = snowplow_utils.base_create_snowplow_events_this_run(
+    sessions_this_run_table=var('snowplow__base_sessions', 'snowplow_base_sessions_this_run'),
+    session_identifiers=var('snowplow__session_identifiers', '[{"schema": "contexts_com_snowplowanalytics_session_identifier_2_0_0", "field": "session_identifier", "prefix": "si_t", "alias": "sito"}, {"schema": "contexts_com_snowplowanalytics_session_identifier_1_0_0", "field" : "session_id", "prefix" : "si_o", "alias": "sido"}]'),
+    session_timestamp=var('snowplow__session_timestamp', 'collector_tstamp'),
```

Comment:
I would maybe not add fallbacks for the variables here, as it is a little confusing/duplication from the project yaml?